views/contact/contactForm.pug

```diff
@@ -0,0 +1,14 @@
+.form-wrapper
+  form.contact-form(action="https://formspree.io/rocha.francisco123@gmail.com" method="POST")
```

Comment:
i know that you can use that action attribute to specify a module to access; i have never seen this before. Won't this be looking for the POST action for that formspree?